ext/opentelemetry-ext-docker-tests/tests/mysql/test_mysql_functional.py

```diff
@@ -74,17 +78,29 @@ def validate_spans(self):
     def test_execute(self):
         """Should create a child span for execute
         """
-        with self._tracer.start_as_current_span("rootSpan"):
-            self._cursor.execute("CREATE TABLE IF NOT EXISTS test (id INT)")
+        try:
```

Comment:
I don't understand this change. The `try` block is also catching an exception raised in `start_as_current_span`. This should not be, we want our tests to fail if there is an exception raised there. If the intention of this change is to catch an `AssertionError` caused by an `Exception` not being raised while running `self._cursor.execute` then the `except` block in line 88 should only be catching `AssertionError` exceptions.

Reply:
Added try inside start_as_current_span, the only problem I can think is if self._cursor.execute patch inside integrations code throws an error this test will not catch it

Reply:
The reasoning behind this change is that we don't care about the actual query to be successful, we only care about Span created correctly when the query is executed